example/AsProject/Logical/ProcessControl/RobotRwsIntf/Main.st

```diff
@@ -0,0 +1,92 @@
+(*
+* File: Cyclic.st
+* Copyright (c) 2023 Loupe
+* https://loupe.team
+* 
+* This file is part of PiperModuleTemplate, licensed under the MIT License.
+*)
+
+PROGRAM _INIT
+	rws.connect.enable := TRUE;
+	rws.connect.configuration.hostname := '192.168.125.1';  //specific per robot used
+	rws.connect.configuration.username := 'Default User';//specific per robot used
+	rws.connect.configuration.password := 'robotics';  //specific per robot used
+	rws.connect.configuration.apiVersion := A3BR_API_VERSION_2;  //Set to Version RWS 2.0 if Robot has a Omnicore processor
+	IF rws.connect.configuration.apiVersion = A3BR_API_VERSION_1 THEN
+		rws.connect.configuration.port := 80;
+	ELSE
+		rws.connect.configuration.port := 443;
+	END_IF
```

Comment:
Nice addition here. Can we point out in a comment that V2 uses HTTPS by default, and that's why the port needs to default to 443?

Reply:
Yes!